### ISSUE TITLE
Is there any solution to ClassFormatException

### ISSUE THREAD

**Issue by johirbuet** [eBay/restcommander#19]
![image](https://cloud.githubusercontent.com/assets/4279025/20652758/1fda00c6-b4c5-11e6-8956-ef719143ce29.png)
I am getting this window again and again. I tried with both mac and windows. I also tried with a number of versions of java starting from 1.7.10. Is there any solution to this problem? Or the current master branch is not actually working?

**Comment by johirbuet** [comment]
These are the exceptions I am getting
**tail -f AgentMaster/logs/system.out**
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        at play.plugins.PluginCollection.loadPlugins(PluginCollection.java:158)
        at play.Play.init(Play.java:294)
        at play.server.Server.main(Server.java:158)
Caused by: org.eclipse.jdt.internal.compiler.classfmt.ClassFormatException
        at org.eclipse.jdt.internal.compiler.classfmt.ClassFileReader.<init>(ClassFileReader.java:372)
        at play.classloading.ApplicationCompiler$1.findType(ApplicationCompiler.java:163)
        ... 48 more
You're running Play! in DEV mode
Could not bind on port 9000

**Comment by jeffpeiyt** [comment]
@johirbuet  thanks for trying REST Commander and I am sorry to hear the issue.

I found some similar error messages:

http://stackoverflow.com/questions/27171440/play-framework-1-2-4-error-while-loading-docviewerplugin

Please help to check if the eclipse loaded the JDK 7 rather tan JDK8 in the settings.

REST Commander may need to upgrade to include a [later play version](https://github.com/playframework/play1) to fix the support with JDK 8. 

We released Parallec.io as the core library for rest commander as the next generation with [several production use cases](https://github.com/eBay/parallec/blob/master/README.md) and [sample servers](https://github.com/eBay/parallec-samples#sample-spark-server)  that would make it very easy to make your application to be rest commander in hours. 

Thanks,

Jeff

**Comment by johirbuet** [comment]
So do you suggest using parallec instead of restcommander?

**Comment by jeffpeiyt** [comment]
@johirbuet  Yes I do suggest using Parallec as we made it with [all the lessons](http://www.parallec.io/#motivation ) in REST Commander. We have used Parallec in production in 4+ teams and places in [eBay and outside](https://medium.com/@jeffpeiyt/a-reusable-part-in-many-cloud-projects-2151f42572e9#.ej1pi1p9b).    I have limited support on REST Commander but it has been internally enhanced and maintained by another team for now.
